backend/src/middleware/validation/validationMiddleware.spec.js

```diff
@@ -119,20 +118,30 @@ beforeEach(async () => {
   moderatingUser = users[1]
   commentingUser = users[2]
   const posts = await Promise.all([
-    factory.create('Post', {
-      id: 'offensive-post',
-      authorId: 'moderating-user',
-    }),
-    factory.create('Post', {
-      id: 'post-4-commenting',
-      authorId: 'commenting-user',
-    }),
+    Factory.build(
+      'post',
+      {
+        id: 'offensive-post',
+      },
+      {
+        authorId: 'moderating-user',
```

Comment:
I know this isn't your code, but it doesn't really make sense, oder?
the person who wrote an offensive post is the moderating user? what they are moderating themselves?